### DISCUSSION TITLE
Unable to contact GSoC mentors

### DISCUSSION THREAD

**Discussion by ken1000minus7** [enviroCar/enviroCar-app#890] [Q&A] [answered]
Hello, I would like to know how I can contact the mentors for information regarding contribution to this project in GSoC. I am unable to join the mailing list or the slack workspace using the links provided on the organization page. 🤔

**Comment by cdhiraj40** [comment, marked as answer]
Hey, @ken1000minus7 Slack is used by them only during the coding period to communicate with our GSoC contributors. At this time, you can contact them from [here](https://wiki.52north.org/Projects/GSoC2022ProjectIdeas). Scroll to the bottom and you will be able to see their emails or for specific project/idea mentors you can see their emails around that idealist  :)

**Reply by ken1000minus7** [reply]
Ok thanks a lot 😄

**Reply by cdhiraj40** [reply]
you can mark this as an answer.
